spatialdata/_core/_spatialdata.py

```diff
@@ -242,12 +280,391 @@ def _init_add_element(self, name: str, element_type: str, overwrite: bool) -> za
+        target_coordinate_system: Optional[str] = None,
+    ) -> None:
+        """
+        Set/replace a transformation of an element, writing it to disk if the SpatialData object is backed.
+
+        Parameters
+        ----------
+        element
+            The element to replace the transformation for.
+        transformation
+            The new transformation.
+        target_coordinate_system
+            The target coordinate system of the transformation. If None, the default coordinate system is used.
+
+        Notes
+        -----
+        - You can also use the static method SpatialData.set_transformation_in_memory() to set the transformation of an
+        element when it is not backed
+        """
+        self.set_transformation_in_memory(element, transformation, target_coordinate_system)
+        self._write_transformations_to_disk(element)
+
+    @staticmethod
+    def remove_transformation_in_memory(
+        element: SpatialElement, target_coordinate_system: Optional[str] = None
+    ) -> None:
+        """
+        Remove a transformation of an element, without writing it to disk.
+
+        Parameters
+        ----------
+        element
+            The element to remove the transformation from.
+        target_coordinate_system
+            The target coordinate system of the transformation to remove. If None, the default coordinate system is used.
+
+        Notes
+        -----
+        - You can also use the method SpatialData.remove_transformation() to remove the transformation of an element when
+        it is backed
+        """
+        from spatialdata._core.core_utils import DEFAULT_COORDINATE_SYSTEM
+
+        if target_coordinate_system is None:
+            target_coordinate_system = DEFAULT_COORDINATE_SYSTEM
+        transformations = _get_transformations(element)
+        assert transformations is not None
+        if target_coordinate_system not in transformations:
+            raise ValueError(f"Transformation to {target_coordinate_system} not found")
+        del transformations[target_coordinate_system]
+        _set_transformations(element, transformations)
+
+    def remove_transformation(self, element: SpatialElement, target_coordinate_system: Optional[str] = None) -> None:
+        """
+        Remove a transformation of an element.
+
+        Parameters
+        ----------
+        element
+            The element to remove the transformation from.
+        target_coordinate_system
+            The target coordinate system of the transformation to remove. If None, the default coordinate system is used.
+
+        Notes
+        -----
+        - You can also use the static method SpatialData.remove_transformation_in_memory() to remove the transformation of
+        an element when it is not backed
+        """
+        self.remove_transformation_in_memory(element, target_coordinate_system)
+        self._write_transformations_to_disk(element)
+
+    @staticmethod
```

Comment:
same again for this, I think there should be 1 method only. Also, when do we need to "remove" transformations>?